h2o-py/tests/testdir_algos/glm/pyunit_link_functions_gaussian_glm.py

```diff
@@ -27,6 +27,7 @@ def link_functions_gaussian():
   print("Create models with canonical link: IDENTITY")
   h2o_model = H2OGeneralizedLinearEstimator(family="gaussian", link="identity",alpha=0.5, Lambda=0)
   h2o_model.train(x=myX, y=myY, training_frame=h2o_data)
+  print(h2o_model)
```

Comment:
This fix should also be in a different PR.

Reply:
Yes, agreed.